### PR TITLE
docs: Replicates changes made to Infura tutorials

### LINKED ISSUE #1469: Replicate tutorial updates
Pass over review as per https://github.com/INFURA/docs/pull/267

### PR DESCRIPTION
**Pull request by m4sterbunny**
# Description
Replicates changes to Infura tutorials

## Issue(s) fixed

#1469 

Fixes 

Tweaks to improve flow and code update to reflect recent library

### CODE REVIEW

File: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md [hunk 1]
```diff
@@ -19,94 +22,135 @@ Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for
 
 ## Steps
 
-### 1. Create a project directory
+### 1. Select your network and verify funds
+
+<Tabs>
+  <TabItem value="View" label="Sepolia" default>
+
+Verify that your wallet has testnet ETH for the Sepolia network. You may use the [Infura faucet](https://www.infura.io/faucet) to top up.
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md:30]: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may

```suggestion
Verify that your wallet has testnet ETH for the Sepolia network. Use the [Infura faucet](https://www.infura.io/faucet) to add funds. 
```

File: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md [hunk 2]
```diff
@@ -19,94 +22,135 @@ Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for
...
+To use an alternative network:
+- Update your [environment file](#4-create-the-env-file) with the alternative network name 
+- Have testnet ETH for that network
+</TabItem>
+</Tabs>
+
+### 2. Create a project directory
 
-Create a new directory:
+Create a new directory for your project. This can be done from the command line:
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md:43]:
```suggestion
Create a new directory for your project:
```

or  

```suggestion
Create a new directory for your project using the command line:
```

File: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md [hunk 3]
```diff
@@ -19,94 +22,135 @@ Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for
...
-### 4. Create `eip1559_tx.js` file
+### 5. Create `eip1559_tx.js` file
 
-In this example, we'll create a JavaScript file (`eip1559_tx.js`) in the project directory which configures and sends the transaction.
+Create a file `eip1559_tx.js` in the project directory which configures and sends the transaction. For example:
 
 :::info
 
-Replace `to_account` with the relevant details.
+Optional: to send test ETH to an account of your choice, update line 15 with your selected account.
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md:120]: I think _optional_  can be removed.

```suggestion
To send test ETH to a specific account, update line 15 with the desired account address.
```

File: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md [hunk 4]
```diff
@@ -19,94 +22,135 @@ Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for
...
-  // Waiting for the transaction to be mined
-  const receipt = await tx.wait();
-  // The transaction is now on chain!
-  console.log(`Mined in block ${receipt.blockNumber}`);
-}
-
-require("dotenv").config();
-main();
+```javascript showLineNumbers
+const { ethers, parseUnits } = require("ethers");
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md:125]: I think code should be formatted for 2 space indentation (for consistency)

File: services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md [hunk 5]
```diff
@@ -19,94 +22,135 @@ Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for
...
+      // Creating and sending the transaction object
+      const tx = await signer.sendTransaction({
+        to: "0x618917c657e9F5b346c0141CB14F5D3CED65D449", // Replace with your selected account
+        value: parseUnits("0.001", "ether"),
+      });
+      console.log("Mining transaction...");
+      console.log(`https://${network}.etherscan.io/tx/${tx.hash}`);
+      // Waiting for the transaction to be mined
+      const receipt = await tx.wait();
+      // The transaction is now on chain!
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/send-a-transaction-ethers.md:146]:
```suggestion
      // The transaction is now on chain
```

File: services/tutorials/ethereum/send-a-transaction/use-web3.js.md [hunk 6]
```diff
@@ -2,29 +2,43 @@
...
 
-[Use the Infura faucet to load testnet ETH](https://www.infura.io/faucet) on your Ethereum account for the Sepolia network.
+Verify that your wallet has testnet ETH for the Sepolia network. You may use the [Infura faucet](https://www.infura.io/faucet) to top up. 
 
-If using a network other than Sepolia, ensure you [update your environment file](#4-create-the-env-file) with the network name.
+</TabItem>
+<TabItem value="Retrieve" label="Alternative network" default>
+
+To use an alternative network:
+- Update your [environment file](#4-create-the-env-file) with the alternative network name
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/use-web3.js.md:37]: I'm not sure, but should this be numbered as a procedure? I know it does not have to ordered, but maybe this is ordinal?

File: services/tutorials/ethereum/send-a-transaction/use-web3.js.md [hunk 7]
```diff
@@ -66,66 +80,78 @@ npm install dotenv --save
...
-
-Never disclose your private key. Anyone with your private keys can steal any assets held in your account.
+```bash
+ETHEREUM_NETWORK = "<network>"
+INFURA_API_KEY = "<API-KEY>"
+SIGNER_PRIVATE_KEY = "<PRIVATE-KEY>"
+```
 
-:::
+- `<NETWORK>` with "sepolia" or the network you are using.
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/use-web3.js.md:94]:
```suggestion
- `<NETWORK>` with `"sepolia"` or the network you are using.
```

File: services/tutorials/ethereum/send-a-transaction/use-web3.js.md [hunk 8]
```diff
@@ -66,66 +80,78 @@ npm install dotenv --save
...
+
 
 ### 5. Create `send.js` file
 
-In this example we'll create a JavaScript file (`send.js`) in the project directory which configures and sends the transaction.
+In this example, we'll create a JavaScript file (`send.js`) in the project directory which configures and sends the transaction.
+
+:::info
+
+Optional: to send test ETH to an account of your choice, update line 20 with your selected account.
```
Comment (joaniefromtheblock) [inline services/tutorials/ethereum/send-a-transaction/use-web3.js.md:122]:
```suggestion
To send test ETH to a specific account, update line 20 with the desired account address.
```

or 

```suggestion
To send test ETH to a specific account, update line 20 with your selected account address.
```

**Review by joaniefromtheblock — DISMISSED** [review]
LGTM. Just a couple of really minor suggestions

**Review by alexandratran — APPROVED** [review]

**Pull request merged**